pkg/cclient/cTest.c

```diff
@@ -0,0 +1,9 @@
+#include "lib_udsclient.h"
+#include <stdio.h>
+
+int main() {
+	printf("C Library: Client Version: %s \n", GetUdsClientVersion());
+	printf("C Library: Server Version: %s \n", GetUdsServerVersion());
+	printf("C Library: Xsk Map FD request: %d \n", RequestXskMapFd("enp94s0f0"));
```

Comment:
I'm not sure about hardcoding this to a physical interface... might be better to create a veth in the context of the pod for testing?